build.gradle

```diff
@@ -97,17 +97,17 @@ subprojects {
       dependency("org.springframework.boot:spring-boot-starter-security:$springBootVersion")
       dependency("org.springframework.boot:spring-boot-starter-thymeleaf:$springBootVersion")
       dependency("org.springframework.boot:spring-boot-starter-web:$springBootVersion")
+      dependency("org.springframework.security.extensions:spring-security-saml2-core:1.0.10.RELEASE")
```

Comment:
also put version in a variable?

Reply:
We are planning to move to SAML implementation which is part of spring-security release train, so I do not think it makes sense anymore.